qbraid_qir/qasm3/oq3_maps.py

```diff
@@ -63,12 +76,25 @@ def u3_gate(builder, theta, phi, lam, qubits):
     # global phase - e^(i*(phi+lambda)/2) is missing in the above implementation
 
 
+def u3_inv_gate(builder, theta, phi, lam, qubits):
```

Comment:
Could you add either Python typehints or a docstring to this function for reusability? At first glance it's not clear how one would invoke it without context clues from where it's used.